code/modules/mob/living/basic/vermin/frog.dm

```diff
@@ -57,15 +66,41 @@
 	)
 	AddElement(/datum/element/connect_loc, loc_connections)
 	AddElement(/datum/element/venomous, poison_type, poison_per_bite)
-	add_cell_sample()
+	AddElement(/datum/element/swabable, frog_cell_line, CELL_VIRUS_TABLE_GENERIC_MOB, 1, 5)
```

Comment:
use the variable you declared? does it even make sense as a variable? probably not